docs/lists_equality_and_diversity.md

```diff
@@ -0,0 +1,28 @@
+## Equality and Diversity
+
+### `DfE::ReferenceData::EqualityAndDiversity::DISABILITIES`
```

Comment:
I think we should refer to this list as "disabilities and health conditions" everywhere, even if that’s more longwinded, as that is how we now refer to the list on Apply, and some of the conditions may not be considered disabilities by some people.

```suggestion
### `DfE::ReferenceData::EqualityAndDiversity::DISABILITIES_AND_HEALTH_CONDITIONS`
```